airflow/config_templates/default_airflow.cfg

```diff
@@ -349,6 +349,14 @@ celery_app_name = airflow.executors.celery_executor
 # your worker box and the nature of your tasks
 worker_concurrency = 16
 
+# The minimum and maximum concurrency that will be used when starting workers with the
+# "airflow worker" command. This defines the number of task instances that
+# a worker will take, minimum numbers are created in beginning and scaled up and down
+# based on number of queued tasks. pick these numbers based on resources on
+# worker box and the nature of the task. If autoscale option is available worker_concurrency
+# will be ignored
+worker_autoscale = 12,16
```

Comment:
We probably shouldn't set a default value for this and worker_concurrency. Please comment this one out.

Reply:
A link to the section of the celery docs about this in the comment would help too.

Reply:
I agree on not using default values but we don't have a provision to use shell functions here so that during execution we can get number of cores and all. What can be done in these scenarios.

Reply:
@ashb this value has been commented out. Please review